mlrose/fitness.py

```diff
@@ -397,13 +397,16 @@ def evaluate(self, state):
         tail_1 = tail(1, state)
 
         # Calculate R(X, T)
-        if (tail_0 > _t and head_1 > _t) or (tail_1 > _t and head_0 > _t):
+        _r = 0
+        _max_score = max(tail_0, head_1)
```

Comment:
I'm not sure whether or not this is the right base evaluation here. I kept it the same as it was before.

Reply:
I'm not 100% certain either, but agree with your decision to keep it the same.